kardia/kardia_test.go

```diff
@@ -0,0 +1,59 @@
+// Package kardia
+package kardia
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/kardiachain/go-kardia/lib/crypto"
+	"go.uber.org/zap"
+
+	"github.com/kardiachain/kardia-explorer-backend/db"
+)
+
+var (
+	dbCfg = db.Config{
+		DbAdapter: "mgo",
+		DbName:    "explorerProd",
+		URL:       "mongodb://kardia.ddns.net:27017",
+		MinConn:   4,
+		MaxConn:   16,
+		FlushDB:   false,
+	}
+	kaiCfg = Config{
+		rpcURL:            []string{"https://dev-1.kardiachain.io"},
+		trustedNodeRPCURL: []string{"https://dev-1.kardiachain.io"},
+	}
+)
+
+func SetupTestAccount() (*ecdsa.PublicKey, *ecdsa.PrivateKey, error) {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `SetupTestAccount` is unused (unused)